tests/jballerina-unit-test/src/test/resources/test-src/annotations/annotations_constant_propagation.bal

```diff
@@ -1,26 +1,30 @@
-import ballerina/http;
+import ballerina/lang.test;
 
-listener http:Listener pathEP = new(9090);
+listener test:MockListener pathEP = new(9090);
 
 const string RESOURCE_PATH = "/{id}";
 const string SERVICE_BASE_PATH = "/hello";
 const string SERVICE_HOST = "b7a.default";
 const string RESOURCE_BODY = "person";
 
-@http:ServiceConfig {
+type resourcePath service object {
```

Comment:
Shall we use an accurate name for this service object? :)

Reply:
Done.